resources/js/Components/PlayerCards/MainPlayerCard.vue

```diff
@@ -90,12 +90,9 @@ const removeFromFavorites = () => {
                     </div>
 
                     <div>
-                        <Link :href="route('player.profile', player.id)">
```

Comment:
I think they should both link to the user profile let's keep the link for both of them